utils/codepointtrie/Cargo.toml

```diff
@@ -35,6 +35,7 @@ all-features = true
 displaydoc = { version = "0.2.3", default-features = false }
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"], optional = true }
 yoke = { version = "0.4.0", path = "../yoke", features = ["derive"] }
+zerofrom = { version = "0.1.0", path = "../zerofrom", features = ["derive"] }
```

Comment:
Question: Should we be depending on `yoke` and `zerofrom` directly, or should we be using the copy re-exported from `icu_provider`?

Reply:
I think either is fine for CPT, i'm using the provider one consistently in components

Reply:
Actually, CPT doesn't currently depend on provider, so it seems best to not introduce a dep.